src/frontends/common/include/openvino/frontend/node_context.hpp

```diff
@@ -48,6 +48,11 @@ class FRONTEND_API NodeContext {
         FRONT_END_NOT_IMPLEMENTED(get_input);
     }
 
+    /// \brief Returns const input with a given index
```

Comment:
/// \brief Returns values from Constant input with the given index as ov::Any. Throws an exception if the input cannot be represented as Constant.

Reply:
done